docs/develop/add-in-manifests.md

```diff
@@ -102,11 +102,14 @@ Only use the [AppDomains](https://dev.office.com/reference/add-ins/manifest/appd
 
 ## Specify domains you want to open in the add-in window
 
-By default, if your add-in tries to go to a URL in a domain other than the domain that hosts the start page (as specified in the [SourceLocation](https://dev.office.com/reference/add-ins/manifest/sourcelocation) element of the manifest file), that URL will open in a new browser window outside the add-in pane of the Office host application. This default behavior protects the user against unexpected page navigation within the add-in pane from embedded **iframe** elements.
+When running in Office Online, your task pane can be navigated to any URL. However, in desktop platforms, by default, if your add-in tries to go to a URL in a domain other than the domain that hosts the start page (as specified in the [SourceLocation](https://dev.office.com/reference/add-ins/manifest/sourcelocation) element of the manifest file), that URL will open in a new browser window outside the add-in pane of the Office host application.
```

Comment:
"However, in desktop platforms, by default," reads a little clunky with all the commas.